### ISSUE TITLE
Updater doesn't detect missing files

### ISSUE THREAD

**Issue by VReaperV** [Unvanquished/updater#137]
Removing any of the files in the game's folder, then starting the updater, does not result in the updater downloading those files, instead it just starts up daemon as is. If daemon executable itself is missing, this also results in a crash.

The above is true at least on Windows, not sure about Linux and Mac versions.

**Comment by DolceTriade** [comment]
Verifying the game on startup takes a long time. I'm not sure if we should create a manifest or something and check against that on every startup?

**Comment by slipher** [comment]
Yeah I say this is a non-goal. 99% of software is not going to recover if you go in and delete a random DLL or data file.

What we can do is detect if the main engine binary is missing and/or detect that the process failed to start, as per #109.

**Comment by DolceTriade** [comment]
At the very least, I think it is a goal to make it easily recoverable *if* we need to repair the game.

**Comment by VReaperV** [comment]
> What we can do is detect if the main engine binary is missing and/or detect that the process failed to start, as per #109.

That sounds reasonable.
